apps/src/util/curriculumRecommender/curriculumRecommenderConstants.js

```diff
@@ -0,0 +1,19 @@
+// Topics we care about promoting more than others
+export const IMPORTANT_TOPICS = [
+  'artificial_intelligence',
+  'physical_computing',
+];
+
+export const UTC_PUBLISHED_DATE_FORMAT = 'YYYY-MM-DD hh:mm:ss UTC';
+
+// Test recommender scoring framework.
+export const FAKE_RECOMMENDER_SCORING = {
```

Comment:
I know this is a bother, but is there any way we could rename this something like TEST_RECOMMENDER_SCORING?

Reply:
Not a bother at all, super open to renames! Yes I like this!